cg/apps/sequencing_metrics_parser/api.py

```diff
@@ -22,15 +24,21 @@ def create_sample_lane_sequencing_metrics_for_flow_cell(
     return create_bcl2fastq_metrics(flow_cell_directory)
 
 
-def create_undetermined_sequencing_metrics_for_flow_cell(
-    flow_cell_directory: Path,
-    bcl_converter: str,
-    non_pooled_lanes_and_samples: List[Tuple[int, str]],
+def create_undetermined_non_pooled_metrics(
+    flow_cell_data: FlowCellDirectoryData,
```

Comment:
Ok, changed it! Just to be clear, this is an instance of `FlowCellDirectoryData` and not the `FlowCell` class. 
Side note: it is not an internal variable since it is part of the function signature